modern/org/commcare/modern/database/TableBuilder.java

```diff
@@ -180,7 +180,7 @@ public String getTableCreateString() {
 
     //sqlite doesn't like dashes
     public static String scrubName(String input) {
-        return input.replace("-", "_");
+        return input.replace("-", "_dash_").replace(".", "_dot_").replace("+", "_plus_");
```

Comment:
Ah nvm, you're saying it's used for column names (not username) I would expect that to fail